material-overrides/home.html

```diff
@@ -43,58 +42,25 @@ <h1>
 
   <div class="feature-cards">
     <div class="feature-card">
-      <h2>Real-time Inference</h2>
+      <h2>Combat AI hallucinations</h2>
       <div class="feature-divider"></div>
-      <p>Instant predictions backed by autoscaling infrastructure. Built for latency-sensitive LLM applications.</p>
-      <a href="/get-started/start-building/real-time/" class="card-button">Use Real-time Inference</a>
+      <p>Use kluster.ai's Verify service to validate the reliability and accuracy of LLM outputs.</p>
+      <a href="/get-started/verify/overview/" class="card-button">Explore Verify</a>
     </div>
 
     <div class="feature-card">
-      <h2>Batch Inference</h2>
+      <h2>Tutorials</h2>
       <div class="feature-divider"></div>
-      <p>Process large datasets with parallel inference at scale. Ideal for precomputing outputs or automating bulk tasks.</p>
-      <a href="/get-started/start-building/batch/" class="card-button">Run Batch Job</a>
+      <p>Follow step-by-step tutorials to build and deploy with kluster.ai's Verify service.</p>
+      <a href="/tutorials/klusterai-api/text-classification/text-classification-openai-api/" class="card-button">Start building</a>
```

Comment:
Just flagging this so we don't forget about it: we'll need to update the link here with the new location of the reliability tutorial